.github/workflows/nodejs.yml

```
@@ -27,3 +27,5 @@ jobs:
     - run: npm run test:all
       env:
         REACT_APP_VALIDATOR: vl.ripple.com
+        REACT_APP_RIPPLED_WS_PORT: 51233
+        REACT_APP_RIPPLED_HOST: fake.ripple.com
```

Is fake.ripple.com just made up? If so, I would prefer to use an `example.com` host (since that's a reserved domain name) so that we don't think there is a mock server or something set up at fake.ripple.com

```suggestion
        REACT_APP_RIPPLED_HOST: fake.rippled.example.com
```